content/faq.md

```diff
@@ -28,4 +48,9 @@ HTTP and HTML allow [streaming pieces of content as soon as possible](https://me
 Sometimes also known as "isomorphic JavaScript", this means using the same components (macros) on both the server and the client.
 
 </dd>
-</dl>
\ No newline at end of file
+</dl>
+
+
+[authors will need a basic understanding of JavaScript]: https://medium.com/@Heydon/reluctant-gatekeeping-the-problem-with-full-stack-e9ad836570f6
```

Comment:
I'd rather avoid Medium, but https://www.heydonworks.com/article/reluctant-gatekeeping-the-problem-with-full-stack is [currently inaccessible](https://twitter.com/heydonworks/status/1216753746979557378). 😞